src/libraries/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -122,10 +124,25 @@ public static void SetConsoleOutputEncoding(Encoding enc)
             if (enc.CodePage != UnicodeCodePage)
             {
                 if (!Interop.Kernel32.SetConsoleOutputCP(enc.CodePage))
-                    throw Win32Marshal.GetExceptionForWin32Error(Marshal.GetLastPInvokeError());
+                {
+                    HandleSetConsoleEncodingError();
+                }
             }
         }
 
+        private static void HandleSetConsoleEncodingError()
+        {
+            var lastError = Marshal.GetLastPInvokeError();
```

Comment:
```suggestion
            int lastError = Marshal.GetLastPInvokeError();
```
The [repo coding conventions](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) do not allow `var` to be used in this position.